itests/org.openhab.binding.wemo.tests/itest.bndrun

```diff
@@ -67,6 +67,7 @@ Fragment-Host: org.openhab.binding.wemo
 	org.eclipse.jetty.websocket.common;version='[9.4.20,9.4.21)',\
 	org.eclipse.jetty.xml;version='[9.4.20,9.4.21)',\
 	org.ops4j.pax.web.pax-web-api;version='[7.2.11,7.2.12)',\
-	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
```

Comment:
You may have removed logback classic, but logback core is still imported at the top.